src/sparsezoo/utils/requests.py

```diff
@@ -106,6 +106,10 @@ def download_get_request(
     if hasattr(args, "release_version") and args.release_version:
         download_args.append(f"release_version={args.release_version}")
 
+    if SPARSEZOO_TEST_MODE:
+        # important, do not remove
+        download_args.append("increment_download=False")
```

Comment:
Another way we could do this in the future is by setting the User-Agent header to something specific to unit tests in this case, and then the server could check against that specific User-Agent. Then we wouldn't need a custom flag!

Reply:
good idea, especially for outside testers /devs, but for now, still need ways to disable download increments for internal use + manual QA runs